contributor-ladder.md

```diff
@@ -0,0 +1,157 @@
+
+### Maintainer
+<!--TODO: project leads to fill in exact details of this role for your project-->
+Description: A Maintainer is a contributor with commit access. Decisions and prioritizes about the project. Maintainer roles may include Community Managers, Project Managers, Release Managers, Docs Managers, and Subproject Lead. An Maintainer must meet the responsibilites and has the requirements of an Reviewer, plus:<!-- sometimes also a project lead - if so, please state in the role defintion -->
+
+* Criteria
+    * Willing to accept additional responsibilites
+    * Ability to keep commitments and follow-up
+    * Demonstrates project leadership
+
+#### Code Maintainer -> expansion pack
+* Criteria
+    * Experience as a contributor
+    * Experience as an reviewer
+* Responsibilities include:
+    * Must follow the [CNCF CoC](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
+    * Merge pull requests in a timely manner
+    * Following the reviewing guide
+    * Helping new contributors
+    * Executing conference/speaking obligations
+    * Handling konveyor relations
+    * Speaking on behalf of the project publically
+* Additional privileges:
+    * Commit access to project repo(s)
+
+## Inactivity
+<!--TODO: project leads to fill in exact details for how you measure inactivity for your project-->
+It is important for contributors to be and stay active to set an example and show committment to the project. Inactivity is harmful to the project as it may lead to unexpected delays, contributor attrition, and a lost of trust in the project. 
+
+* Inactivity is measured by:
+    * Periods of no contributions for longer than X months
+    * Periods of no communication for longer than X months
+* Consequences of being inactive include:
+    * Involuntary removal or demotion
+    * Being asked to move to Emeritus status
+
+## Involuntary Removal or Demotion
+<!-- project leads may want to consider integrating this section under every role description -->
+
+Involuntary removal/demotion of a contributor happens when responsibilites and requirements aren't being met. This may include repeated pattern of inactivity, extended period of inactivity, and/or a violation of the Code of Conduct. This process is important because it protects the community and its deliverables while also opens up opportunities for new contributors to step in.
+
+Involuntary removal or demotion is handled by the community governence committe. 
+
+## Stepping Down/Emeritus Process
+If and when contributors' committment levels change, contributors can consider stepping down (moving down the contributor ladder) vs moving to emeritus status (completely stepping away from the project). 
+
+Please reach out to the associated project lead(s) to discuss this process.
+
+## Contacts
+* For community needs/issues please reach out to:
+    *  James Labocki - jlabocki@redhat.com
+    *  Brett Thurber - bthurber@redhat.com
+* Project leads:
```

Comment:
I'm wondering whether the leads should not only be limited to Engineering resources and include those with a vision of the strategic direction e.g. Miguel Perez Collino for Forklift, Ramon Roman Nissen for Tackle, etc.

Reply:
Makes sense to add.